tox.ini

```
@@ -179,7 +179,7 @@ commands=
 [testenv:func-mitm-py{37,38,39,310}]
 install_command=pip install --extra-index-url https://download.pytorch.org/whl/cpu {opts} {packages}
 deps =
-    yea-wandb=={env:YEA_WANDB_VERSION}
+    yea-wandb==https://github.com/wandb/yea-wandb/archive/estella/ttlYea.zip
```

```suggestion
    https://github.com/wandb/yea-wandb/archive/estella/ttlYea.zip
```
same on line 161 and 199